test/tsv_to_mrmatrix_test.py

```diff
@@ -143,9 +158,126 @@ def test_parse(self):
             # https://github.com/higlass/clodius/issues/62
             res_2 = hdf5['resolutions']['2']['values']
             assert_array_equal(res_2[0], [0] * 256)
-            assert_array_equal(res_2[1], [2] * 256) # Stradles the 0 and 1 rows
+            assert_array_equal(res_2[1], [2] * 256)
+            # Stradles the 0 and 1 rows
             assert_array_equal(res_2[2], [4] * 256)
-            assert_array_equal(res_2[3], [0] * 256) # -1 and +1 cancel out
+            assert_array_equal(res_2[3], [0] * 256)
+            # -1 and +1 cancel out
             assert_array_equal(res_2[4], [0] * 256)
             assert_array_equal(res_2[5], [0] * 256)
             assert_array_equal(res_2[6], [0] * 256)
+            # TODO: We lose nan at higher aggregations:
+            # Maybe regular mean/sum instead of treating missing values as 0?
```

Comment:
How do you mean? Isn't the opposite problem true if we use regular sum instead of `nansum`?

Reply:
Ignore the "Maybe regular mean/sum", but otherwise this is the issue in https://github.com/higlass/clodius/issues/62: Either we don't have the nan counts at higher aggregation levels, or I'm confused.

Reply:
(More info at #62)